### PR TITLE
make temporary get involved page

### LINKED ISSUE #47: Embed simple donation widget
Code:
```
<!-- Begin Give Lively Fundraising Widget -->
<script>gl=document.createElement('script');gl.src='https://secure.givelively.org/widgets/simple_donation/bit-by-bit-coding.js?show_suggested_amount_buttons=true&show_in_honor_of=true&address_required=false&has_required_custom_question=null&suggested_donation_amounts[]=10&suggested_donation_amounts[]=35&suggested_donation_amounts[]=70&suggested_donation_amounts[]=140';document.getElementsByTagName('head')[0].appendChild(gl);</script><div id="give-lively-widget" class="gl-simple-donation-widget"></div>
<!-- End Give Lively Fundraising Widget -->
```

### PR DESCRIPTION
**Pull request by sophiaaaaa**
the embedded google form is awkward (especially on mobile) but fixes #47

### CODE REVIEW

File: getinvolved.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+                position on your resume while you’re at it.
+            </p>
+            <p>
+                Contact us at <a href="mailto:{{site.contact.email}}">{{site.contact.email}}</a> for more info.
+            </p>
+        </div>
+        <a href="https://docs.google.com/forms/d/e/1FAIpQLSeapmj33WC0HLmJRsl2ICq3KTplADuCi2BM97jbV2GSQcP1TA/viewform?usp=sf_link" class="signup-wrap embedfallbackbutton">
+                <button class="action-button">Volunteer</button>
+            </a>
+            <iframe id="embeddedform" src="https://docs.google.com/forms/d/e/1FAIpQLSeapmj33WC0HLmJRsl2ICq3KTplADuCi2BM97jbV2GSQcP1TA/viewform?embedded=true" width="640" height="1443" frameborder="0" marginheight="0" marginwidth="0">Loading…</iframe>
```
Comment (MoralCode) [inline getinvolved.html:44]: Do we want to keep the google form embedded? IMO i think we could probably change it back to a button that says like "volunteer here" or something, although this is open to discussion

File: styles/style.scss [hunk 2]
```diff
@@ -48,13 +48,18 @@ h2 {
     font-family: $heading-font;
     font-weight: 700;
     margin-bottom: 2%;
+    text-align: left;
```
Comment (MoralCode) [inline styles/style.scss:51]: were all `h2`'s on the site not already left-aligned?

File: styles/style.scss [hunk 3]
```diff
@@ -231,14 +236,15 @@ img {
...
-    // override the defailt width and height HTML attributes
+    // override the default width and height HTML attributes
     width: 100%;
     height: 100%;
 }
 
 @media (min-width: 768px) {
     .details-container{
-        flex-direction: row;
+        display: flex;
```
Comment (MoralCode) [inline styles/style.scss:246]: I'm not sure this is necessary. the `details-container` class already has the `display: flex`, and `flex-wrap` properties. this is just a media query to change it from `flex-direction: column;` to `flex-direction: row;` on devices bigger than `768px`

File: styles/style.scss (hunk 2 above)
Comment (sophiaaaaa) [reply styles/style.scss:51]: oops I think there was a centertext class added before, I forgot to delete the text-align after I realized

File: getinvolved.html (hunk 1 above)
Comment (lucas-28) [reply getinvolved.html:44]: I think it would look cleaner to have a button instead.

Comment (nirapatelm) [reply getinvolved.html:44]: > Do we want to keep the google form embedded? IMO i think we could probably change it back to a button that says like "volunteer here" or something, although this is open to discussion

lets talk about that during standup today.

Comment (MoralCode) [reply getinvolved.html:44]: i think selena mentioned that thi is supposed to be a super hacky/temporary thing so we can prob wait to fix it in the real version

**Comment by MoralCode** [comment]
closing as a duplicate of #66

**Pull request closed without merging**